docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/01_backwards_compatibility.md

```diff
@@ -22,7 +22,27 @@ While we only have one extension published, there are already several DApp imple
 This will address the case when the extension speaks the newer version, and we will try to avoid the opposite case by releasing the upgraded Sporran much earlier.
 We will not handle the case when the user manually disabled Sporran updates.
 
-Sporran should achieve compatibility by translating the messages sent to the previous versions of DApps.
+Sporran should achieve compatibility by translating the messages received from and sent to the previous versions of DApps.
+
+For `submit-terms` replace the items of the `cTypes` content property with the values of their `schema` properties.
+
+```ts
+// before
+{
+  cTypes: Array<{
+    schema: ICTypeSchema
```

Comment:
ICTypeSchema no longer exists, do you think that's a problem?

Reply:
This is in the "before" section, I guess people will be able to figure it out. We also might address that when rewriting the text for external consumption.